src/Event/EncryptingFields.php

```diff
@@ -0,0 +1,11 @@
+<?php
+declare(strict_types=1);
+
+namespace MyOnlineStore\EventSourcing\Event;
+
+/**
+ * @extends \ArrayObject<array-key, string>
```

Comment:
With the introduction of static checks on array type hints, such overhead of wrapping a simple string array to a "domain" specific collections seems unnecessary